infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -504,6 +504,151 @@ all the operations for the connector:
+
+Choose a site in your Jira workspace to allow access for this application and click on `Accept`.
+
+As the callback is not existing, you will see an error. But in the URL of your browser you will see something like this as URL:
+
+`http://localhost/?state=YOUR_USER_BOUND_VALUE&code=eyJhbGc...`
+
+The content of the `code` parameter is the `authentication code` required for next step.
+
+**NOTE** This `Authorization Code` if for a one single use; if expired or used you will need to get it again pasting the URL in the browser.
+5. Now, replace the values in following URL and run it from command line in your terminal. Replace `YOUR_AUTHENTICATION_CODE`, `YOUR_CLIENT_ID` and `YOUR_CLIENT_SECRET` in the placeholders:
+
+`curl --request POST --url 'https://auth.atlassian.com/oauth/token' --header 'Content-Type: application/json' --data '{"grant_type": "authorization_code","client_id": "YOUR_CLIENT_ID","client_secret": "YOUR_CLIENT_SECRET", "code": "YOUR_AUTHENTICATION_CODE", "redirect_uri": "http://localhost"}'`
+
+6. After running that command, if successful you will see a [JSON response](https://developer.atlassian.com/cloud/jira/platform/oauth-2-3lo-apps/#2--exchange-authorization-code-for-access-token) like this:
+
+```json
+{
+    "access_token": "some short live access token",
+    "expires_in": 3600,
+    "token_type": "Bearer",
+    "refresh_token": "some long live token we are going to use",
+    "scope": "read:jira-work offline_access read:jira-user"
+}
+```
+7. You will need to provide `cloudId` parameter of your Jira instance. To retrieve it, please run the following command replacing adding the
+`ACCESS_TOKEN` obtained in the previous step:
+
+ `curl --header 'Authorization: Bearer <ACCESS_TOKEN>' --url 'https://api.atlassian.com/oauth/token/accessible-resources'`
+
+And its response will be something like:
+
+```json
+[
+  {
+  "id":"SOME UUID",
+  "url":"https://your-site.atlassian.net",
+  "name":"your-site-name",
+  "scopes":["read:jira-user","read:jira-work"],
+  "avatarUrl":"https://site-admin-avatar-cdn.prod.public.atl-paas.net/avatars/240/rocket.png"
+  }
+]
+```
+
+Use that id as `jira_cloud_id` parameter to include as part of Terraform deployment. That will target your instance for REST API requests.
+8. Finally set following variables in AWS System Manager parameters store / GCP Cloud Secrets (if default implementation):
+  - `PSOXY_JIRA_CLOUD_ACCESS_TOKEN` secret variable with value of `access_token` received in previous response
+  - `PSOXY_JIRA_CLOUD_REFRESH_TOKEN` secret variable with value of `refresh_token` received in previous response
+  - `PSOXY_JIRA_CLOUD_CLIENT_ID` with `Client Id` value.
+  - `PSOXY_JIRA_CLOUD_CLIENT_SECRET` with `Client Secret` value.
+
```

Comment:
please also put version of this in `docs/sources/atlassian/jira.md` or something

Reply:
Yeah I was to do it in next PR, but included here.